src/Request.php

```diff
@@ -110,6 +115,18 @@ public function getPayload(string $key, mixed $default = null): mixed
         return $payload[$key] ?? $default;
     }
 
+    /**
+     * Get raw payload
+     *
+     * Method for getting the HTTP request payload as a raw string.
+     *
+     * @return string
+     */
+    public function getRawPayload(): string
+    {
+        return $this->raw;
```

Comment:
You need to ensure `generateInput()` runs before you can access this property, I think you can just call it before returning the data.